data/hai outfits.txt

```diff
@@ -600,3 +600,12 @@ outfit `"Bufaer" Atomic Steering`
 	"turning energy" 5.2
 	"turning heat" 13.8
 	description "Nine out of ten human captains interviewed in Hai space say that they would choose Hai atomic engines over those of Deep Sky any day... that is, if they could afford such expensive engines."
+
+outfit "Pulse Rifle"
+	category "Hand to Hand"
+	cost 11000
+	thumbnail "outfit/hai rifle"
+	"capture attack" .8
+	"capture defense" 1.3
+	"unplunderable" 1
+	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to not be lethal outright, if several bolts hit a target rapidly, it will shut down the nervous system, killing the target."
```

Comment:
Maybe reword to  `"The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to more effectively incapacitate its target, if several bolts land in quick succession, the accumulated charge has a lethal effect on most beings' nervous systems."`

Also, "Pulse Rifle" or "Ion Pulse Rifle"?

Reply:
its the pulse rifle since its just been adjusted the original one was made to be lethal and has since been adjusted to the more passive weapon you see now